### PR TITLE
fixed the issue with accessors for constant state variables

### PR DESCRIPTION
**Pull request by LianaHus**
fixed the issue with accessors for constant state variables

### CODE REVIEW

**Comment by LianaHus** [comment]
https://github.com/ethereum/solidity/issues/16

File: libsolidity/AST.cpp [hunk 1]
```diff
@@ -554,8 +554,11 @@ void VariableDeclaration::checkTypeRequirements()
 	{
 		if (!dynamic_cast<ContractDefinition const*>(getScope()))
 			BOOST_THROW_EXCEPTION(createTypeError("Illegal use of \"constant\" specifier."));
-		if ((m_type && !m_type->isValueType()) || !m_value)
-			BOOST_THROW_EXCEPTION(createTypeError("Unitialized \"constant\" variable."));
+		if (!m_value)
+			BOOST_THROW_EXCEPTION(createTypeError("Uninitialized \"constant\" variable."));
+		else if (!m_type->isValueType())
```
Comment (chriseth) [inline libsolidity/AST.cpp]: Why did you remove `m_type && ...`?

Comment (LianaHus) [inline libsolidity/AST.cpp]: by mistake. I'll create another PR

Comment (LianaHus) [inline libsolidity/AST.cpp]: https://github.com/ethereum/solidity/pull/24
